azure-devops/app/05_tlscert-weuuat-wallet-internal-uat-platform-pagopa-it.tf

```diff
@@ -0,0 +1,93 @@
+variable "tlscert-weuuat-wallet-internal-uat-platform-pagopa-it" {
```

Comment:
internal certificate pipelines better to move into the domain folder

Reply:
ok no problem.